CMakeLists.txt

```diff
@@ -12,51 +12,49 @@ include(FetchContent)
 
 set(TARGETS_EXPORT_NAME ${CMAKE_PROJECT_NAME}Targets)
 
-# Build the tests only if enabled via the CLI flag: BUILD_TESTING.
-if(BUILD_TESTING)
+option(
+    OPTIONAL26_ENABLE_TESTING
+    "Enable building tests and test infrastructure"
+    ${PROJECT_IS_TOP_LEVEL}
+)
+
+# Build the tests if enabled via the option OPTIONAL26_ENABLE_TESTING
```

Comment:
Can you remind me why BUILD_TESTING is not enough/OK ?

The Beman Standard specify to have `BUILD_TESTING`.

Reply:
It doesn't compose. Best practice for Cmake is to prefix options with the project they apply to, and to default the value to true for a top level project. 

Patten documented in Professional CMake.

Reply:
Agree that it's the best practice from CMake docs, but I would expect to first make a PR in the Beman Standard.

Reply:
Fortunately, it's a recommendation, but I'll PR that, now that I've seen it. But since this actually breaks trying to compose this into another beman project, it would be nice to fix it now.

Reply:
@JeffGarland should we make it a requirement?